src/config/logger.ts

```diff
@@ -56,34 +38,38 @@ const errorSerializer = (err) => {
 	};
 };
 
-const hashSerializer = (data?: string): string | undefined => {
-	if (!data || !isString(data)) {
-		return undefined;
-	}
-	return createHashWithSharedSecret(data);
-};
-
 export const defaultLogLevel: LogLevel = process.env.LOG_LEVEL as LogLevel || "info";
 
+const loggerStream = (safe = true): Logger.Stream => ({
+	type: "raw",
+	stream: safe ? new SafeRawLogStream(FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME) : new UnsafeRawLogStream(FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME),
+	closeOnExit: false
+});
```

Comment:
Not sure I'm a fan of this.  I'd probably just create 2 constants, one for a safe log stream and one unsafe, then add those both to the logger.  Unless there's a reason for this?

Reply:
I agree it is a bit messy. Have split it out.